docs/data/sdks/sdk-comparison.md

```diff
@@ -0,0 +1,81 @@
+# All SDKs comparision 
+
+This article provides a comparison between different SDKs across platforms. This overview offers key insights for developers to make informed decisions when selecting the appropriate SDK.
+
+!!!note
+
+    While there are multiple versions of our SDK available, we highly recommend using the latest version rather than maintenance SDKs. Maintenance versions only provide support and will eventually be deprecated. The [latest SDK](./sdk-architecture.md) adopts an aligned architecture, with no extra dependencies, and offers enhanced stability for optimal performance.
+
+## Browser
+
+| Feature| <div class="big-column"> [Latest Browser SDK](../typescript-browser/) </div>| <div class="big-column">[Latest Browser SDK < v1.9](../typescript-browser/)</div>|  <div class="big-column"> [Marketing analytics Browser](../marketing-analytics-browser/) <div> |  <div class="big-column"> [Mantiance Browser SDK](../javascript/) </div> |
```

Comment:
Couple thoughts to make the size of this table more manageable, and the page more scannable.
1. What about combining "Latest Browser SDK" and "Latest Browser SDK < v1.9" columns? We could then only include the differences in < 1.9 underneath the "latest" info.
2. Change the features names to be headers (i.e. H3/H4) and then have a table under each feature comparing the different SDKs. This would make the features browsable in the page outline (on the right of the page) and give more space for table content. This would be similar to the 
"Default Web Attribution vs using webAttributionPlugin" section below.

Reply:
I like the second part. 

For the first part. I also feel 4 columns also a little bit crowded. How do you like to compare everything with the latest SDK. We can combine Latest Browser SDK" and "Latest Browser SDK < v1.9 like what you said.
<img width="1066" alt="Screen Shot 2023-05-09 at 3 17 40 PM" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/13028329/bcaf0128-de71-4071-8094-8414682ef60d">